package/cfg/k3s-cis-1.7-hardened/node.yaml

```diff
@@ -317,14 +315,14 @@ groups:
           By default, K3s sets the event-qps to 0. Should you wish to change this,
           If using the K3s config file /etc/rancher/k3s/config.yaml, set the following parameter to an appropriate value.
           kubelet-arg:
-          - "event-qps=<value>"
+            - "event-qps=<value>"
           If using the command line, run K3s with --kubelet-arg="event-qps=<value>".
           Based on your system, restart the k3s service. For example,
           systemctl restart k3s.service
         scored: false
 
       - id: 4.2.9
-        text: "Ensure that the --tls-cert-file and --tls-private-key-file arguments are set as appropriate (Manual)"
+        text: "Ensure that the --tls-cert-file and --tls-private-key-file arguments are set as appropriate (Automated)"
```

Comment:
Together with Automated, should this not be either `skip: true` or `scored: true`?

Reply:
This is set to `scored: false` because its a permissive version of the scan. Not the Remediation message that was added with https://github.com/rancher/security-scan/pull/135